app.ts

```diff
@@ -42,10 +42,12 @@ app.post('/interactions', (req: Request<Record<string, unknown>, Record<string,
    * See https://discord.com/developers/docs/interactions/application-commands#slash-commands
    */
   if (type === InteractionType.APPLICATION_COMMAND) {
-    const { name } = data;
+    const command = data.command;

```

Comment:
Name is a specific field of the ApplicationCommand model. ref - https://discord.com/developers/docs/interactions/application-commands